wikimedia/wikipedia-ios
# Image recommendations - Article Intro View UI

**Phabricator:** 
https://phabricator.wikimedia.org/T356816 (UI)

### Notes
This PR adds UI for the article intro view.

This PR is dependent on #4760.

### Test Steps
1. Set primary language to a language with image recommendations. Lately I've been using FR Wikipedia.
2. Tap Explore feed notifications center icon.
3. Confirm UI looks okay.

### Screenshots

![screenshot](https://github.com/wikimedia/wikipedia-ios/assets/3620196/7e3bf6ae-cfad-4851-b949-e75a570b70ed)
